src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModulePropertiesController.cs

```diff
@@ -149,8 +152,19 @@ public async Task<ActionResult> SaveProperty([FromBody]Property property)
             await _propertyService.SaveChangesAsync(new[] { property });
 
             return NoContent();
+        }
+
+        /// <summary>
+        /// Validate name for Product-level (unmanaged) property
+        /// </summary>
+        [HttpPost]
+        [Route("validatename")]
```

Comment:
https://restfulapi.net/resource-naming/
```suggestion
        [Route("validate-name")]
```